README.md

```diff
@@ -1 +1,177 @@
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSLQ</a></li>
+    
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Using RoR to structure the project.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Make sure you have Ruby and Ruby on Rails installed on your computer before running the project. To check if you have Ruby installed, open your console and run the command ruby -v.
+
+If Ruby is not installed, follow the instructions below based on your operating system:
+
+Windows
+Follow the [tutorial](https://gorails.com/setup/windows/10) for installing Ruby on Windows.
+
+Linux
+Run the following commands to install Ruby in linux mint:
+  # Install Ruby
+sudo apt update
+sudo apt install ruby
+  ruby -v
+```
+
+To install Ruby on Rails, run:
+
+```sh
+  gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Javed-Shahbaz/Blog-App.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install 
+```
+
+### Usage
+
+```sh
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+### Deployment
+To deploy the project locally, run the following command:
+
+```sh
+  rails s
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Javed Shahbaz**
+
+- GitHub: [@Javed-Shahbaz](https://github.com/Javed-Shahbaz)
+- Twitter: [@javedshahbaz007](https://twitter.com/javedshahbaz007)
+- LinkedIn: [@javedshahbaz](https://www.linkedin.com/in/javedshahbaz/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Developing the UI]**
+- [ ] **[Deploying the site]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Javed-Shahbaz/Blog-App/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please support me to improve my skills. Give a star if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to everyone who watch my code and like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License.md) licensed.
+
```

Comment:
- Kindly, the `mit` link is broken, please fix it to have a professional readme 👍